app/src/main/res/xml/about.xml

```diff
@@ -139,5 +139,10 @@
             avatar="https://avatars3.githubusercontent.com/u/9938439"
             locales="fr"
             login="yassineim" />
+        <me.jfenn.attribouter.wedges.TranslatorWedge
+            name="一座桥在水上"
+            avatar="https://avatars3.githubusercontent.com/u/30341059"
+            locales="zh-rCN"
```

Comment:
I should have mentioned that the [Attribouter](https://jfenn.me/projects/attribouter/wiki/TranslatorWedge) library that I use for the About page uses Java's [`Locale.getISOLanguages()`](https://docs.oracle.com/javase/7/docs/api/java/util/Locale.html#getISOLanguages()), which only supports 2-letter language codes.

You should use just `zh` there, otherwise it's not going to say "Chinese" in the about page.

Can you please change it? Thank you!

Reply:
Okey, I changed it.
◉‿◉